scripts/clap.bash

```diff
@@ -0,0 +1,168 @@
+# clap - a BASH argument parser
```

Comment:
Note: This is a stock bash argument parser, used in nns-dapp and other repos.  It is copied here verbatim from the nns-dapp repo.  See the other file for usage.